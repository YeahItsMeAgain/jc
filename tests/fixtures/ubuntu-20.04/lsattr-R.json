[{"file": "/tmp/folder/folder", "Extents": true}, {"file": "/tmp/folder/folder/test_file", "Compression_Requested": true, "Extents": true}]