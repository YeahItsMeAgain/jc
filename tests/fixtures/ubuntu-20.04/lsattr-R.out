--------------e----- /tmp/folder/folder

/tmp/folder/folder:
--------c-----e----- /tmp/folder/folder/test_file

